<launch>

<node name="mic_logger" pkg="mic_logger" type="MicLogger.py" output="screen">
    <param name="com_port" type="string" value="/dev/ttyACM0" /> 
    <param name="com_baud" type="int" value="4000000" /> 
    <param name="com_cts" type="bool" value="true" /> 

    <rosparam param="adc_delim_seq">[255,255,255,255,255]</rosparam> <!-- list of byte values 0-255 -->

    <param name="pga_gain" type="double" value="157" /> <!-- options: 1, 10, 20, 30, 40, 60, 80, 120, 157, 0.25 -->
    <param name="pga_offset" type="double" value="12.7" /> <!-- options: 0.0, 1.3, 2.5, 3.8, 4.9, 6.1, 7.3, 8.4, 10.6, 11.7, 12.7, 13.7, 14.7, 15.7, 16.7, 17.6 -->
    <param name="adc_channel" type="int" value="1" /> <!-- options: 0, 1, 2 -->

    <param name="gain_adjustment_period" type="double" value="0.0" /> <!-- negative for disable, 0 for immediate, positive is period in seconds -->
    <param name="settings_print_period" type="double" value="1.0" /> <!-- negative/zero for disable, positive is period in seconds -->

    <param name="adc_freq" type="int" value="160000" /> 
    <param name="adc_samples_per_packet" type="int" value="500" /> 
</node>

</launch>
